recipe-box-frontend/src/main/frontend/src/components/Header/Header.css

```diff
@@ -38,6 +38,10 @@
     height: 25px;
 }
 
+.profile-logo-image {
+    height: 25px;
+}
+
```

Comment:
Pro tip: You should use Optical Compensation here: make the circle's diameter 5% to 10% bigger than the addRecipe's button's height.

Also, the two elements are too close to each other. Since they are not related in function, they should feel a bit more separated.